changelog.md

```diff
@@ -1,5 +1,10 @@
 ## Changes to 1.8.xxx
 
+### Upgrades only from Marathon 1.6+
+
+You can only upgrade to Marathon 1.8 from 1.6.x and 1.7.x. If you'd like to upgrade from an earlier version you should
```

Comment:
Did we ever properly describe Marathon upgrade strategy? Feels like we should spend a few more words on it.

Reply:
You mean the process of upgrading or the strategy? We have [dcos](https://mesosphere.github.io/marathon/docs/upgrade/index.html). There are a little old but should still apply.